config/settings/docker.py

```diff
@@ -63,7 +64,7 @@
         'NAME': POSTGRES_DB, # noqa F405
         'USER': POSTGRES_USER, # noqa F405
         'PASSWORD': POSTGRES_PASSWORD, # noqa F405
-        'HOST': "db-postgres",
+        'HOST': (POSTGRES_HOST if 'POSTGRES_HOST' in os.environ else "db-postgres"), # noqa F405
```

Comment:
I think the below text is a bit more concise
```suggestion
        'HOST': os.environ.get('POSTGRES_HOST', 'db-postgres'), # noqa F405
```